src/sentry/api/endpoints/relay_register.py

```diff
@@ -42,13 +43,10 @@ def is_internal_relay(request, public_key):
     """
     Checks if the relay is allowed to register, otherwise raises an exception
     """
-    if (
-        settings.DEBUG
-        or request.META.get("REMOTE_ADDR", None) in settings.INTERNAL_IPS
-        or public_key in settings.SENTRY_RELAY_WHITELIST_PK
-    ):
+    if settings.DEBUG or public_key in settings.SENTRY_RELAY_WHITELIST_PK:
         return True
-    return False
+
+    return is_internal_ip(is_internal_ip)
```

Comment:
`is_internal_ip` uses [`INTERNAL_SYSTEM_IPS`](https://github.com/getsentry/sentry/blob/bfe27f43dc28d308546ad03446e24c1ebf9b6c9a/src/sentry/conf/server.py#L83-L99). Is that what you want?

You might need to add the docker network to that list in the onpremise config.

Reply:
Already done: [`9885965` (#572)](https://github.com/getsentry/onpremise/pull/572/commits/988596581bac49f9cbab283a2fa5b1e8c1de7ade)